mix.exs

```diff
@@ -25,8 +25,8 @@ defmodule Explorer.MixProject do
   defp deps do
     [
       {:ex_doc, "~> 0.24", only: :dev, runtime: false},
-      {:nx, "~> 0.1.0-dev", github: "elixir-nx/nx", branch: "main", sparse: "nx"},
-      {:rustler, "~> 0.22.0"}
+      {:nx, "~> 0.1.0"},
+      {:rustler, "~> 0.23.0"}
```

Comment:
A colleague at work couldn't run on M1 using rustler 0.22 and had to use `override: true` with 0.23. Let me know if this is problematic in any way.